content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -143,9 +143,9 @@ The prefix used for Prometheus metrics from the query controller has changed fro
 - Display task IDs in the tasks list in the InfluxDB UI.
 - Write to standard output (`stdout`) when `--output-path -` is passed to [`influxd inspect export-lp`](/influxdb/v2.0/reference/cli/influxd/inspect/export-lp/).
 - Add `-p, --profilers` flag to [`influx query` command](/influxdb/v2.0/reference/cli/influx/query/)
-  to enable [Flux profilers](/influxdb/v2.0/reference/flux/stdlib/profiler/) on
+  to enable [Flux profilers](/{{< latest "flux" >}}/stdlib/profiler/) on
   a query executed from the `influx` CLI.
-- Update InfluxDB OSS UI to match InfluxDB Cloud.
+- Update InfluxDB OSS UI to match InfluxDB Cloud./influxdb/v2.0/write-data/load-data/
```

Comment:
Update InfluxDB OSS UI to match InfluxDB Cloud. /influxdb/v2.0/write-data/load-data/